### ISSUE TITLE
Getter and setter? Decorating accessors

### ISSUE THREAD

**Issue by topaxi** [rwjblue/ember-computed-decorators#5]
I expected this to work more along the lines of

```javascript
export default Ember.Component.extend({
  @computed('first', 'last')
  get name(first, last) {
    return `${first} ${last}`;
  },

  @computed('first', 'last')
  set name(value) {
    let [ first, last ] = value.split(' ')
    this.set('first', first)
    this.set('last', last)
    return value
  }
})
```

What do you think?

**Comment by rwjblue** [comment]
There was a reason that I didn't do this via the ES6 getter/setter syntax, but I can't recall exactly why.  I'll reinvestigate to see what the issue was.

---

I added a test that allows usage of Ember's setter/getter CP's: https://github.com/rwjblue/ember-computed-decorators/blob/master/tests/unit/computed-test.js#L85-L122, but marked it as `skip` because babel does not currently allow decorating non-functions. I am waiting on confirmation from @wycats before opening an issue for it with the babel folks.

**Comment by cibernox** [comment]
@rwjblue Does it make sense to apply this decorators to ES5 getter/setter functions?

Example:

```js
let obj = {
  first: 'rob',
  last: 'jackson',

  /* jshint ignore:start */
  @computed('first', 'last')
  /* jshint ignore:end */
  get name() {
  },

  set name(value, first, last) {
  }
};
```

**Comment by rwjblue** [comment]
@cibernox - As I mentioned above, it doesn't work but I can't recall why (I do remember trying).  I'll play with it more tonight.

Also, that commented out test uses something like:

```javascript
let obj = {
  first: 'rob',
  last: 'jackson',

  @computed('first', 'last')
  name: {
    set() { },
    get() { }
  }
};
```

But babel is getting confused with decorators on non-functions.

**Comment by rwjblue** [comment]
I am crazy, @stefanpenner implemented get/set in #8.

**Comment by stefanpenner** [comment]
> What do you think?

we will support this eventually :)
